extensions/src/main/resources/org/pentaho/platform/plugin/services/messages/messages.properties

```
@@ -411,7 +411,7 @@ ZIPFILE.ExceptionOccurred=Exception occurred when processing Zip:
 
 UserRoleDaoService.PassValidationError_Length=have at least {0} characters
 UserRoleDaoService.PassValidationError_SpecChar=have at least one special character
-UserRoleDaoService.PassValidationError_PermittedChars=only contain the following characters: a-z A-Z 0-9 _ . , : ; < > | ! @ # $ % ^ & * ( ) [ ]
+UserRoleDaoService.PassValidationError_PermittedChars=only contain the following characters: a-z A-Z 0-9 _ . , : ; < > | ! @ # $ % ^ & * ( ) [ ] -
```

Use the `ALLOWED_CHARS_LIST` constant instead of repeating this string.

```suggestion
UserRoleDaoService.PassValidationError_PermittedChars=only contain the following characters: {0}
```